SW updates including reference to Github example and fixed links to e… (#4049)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/serverless/chap-kogito-orchestrating-serverless.adoc

.Structure of the example `ordersworkflow.sw.json` workflow definition
image::kogito/serverless/serverless-workflow-kogito-orders-structure.png[Image of `ordersworkflow.sw.json` workflow definition]

> The `HandleNewOrder` state is the workflow starting state. This state is an `event` type state that requires one or more events to be consumed before the workflow can be invoked. This event state defines the instances of this workflow that are created when the referenced `NewOrderEvent` is consumed. In this case, events are consumed via Kafka.

Reviewer: ```suggestion
The `HandleNewOrder` state is the workflow starting state. This state is an `event` type state that requires one or more events to be consumed before the workflow can be invoked. This event state defines the instances of this workflow that are created when the referenced `NewOrderEvent` is consumed. In this case, events are consumed using Kafka.
```

For more {PRODUCT} examples that use Serverless Workflow, see the following example applications in GitHub:

* https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-events-quarkus[`serverless-workflow-events-quarkus`]: A Serverless Workflow service for processing job applicant approvals and that showcases event-driven services
* https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-greeting-quarkus[`serverless-workflow-greeting-quarkus`] and https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-greeting-springboot[`serverless-workflow-greeting-springboot`]: A Serverless Workflow greeting service with both JSON and YAML workflow definitions
* https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-service-calls-quarkus[`serverless-workflow-service-calls-quarkus`] and https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-service-calls-springboot[`serverless-workflow-service-calls-springboot`]: A Serverless Workflow service for returning country information

> * https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-github-showcase[`serverless-workflow-github-showcase`]: In this example we will deploy a GitHub "bot" application that will react upon a new PR being opened in a given GitHub project. The "bot" is implemented via service and event orchestration approach with Kogito implementation of the Serverless Workflow specification.

Reviewer: ```suggestion
* https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-github-showcase[`serverless-workflow-github-showcase`]: In this example, a Github bot application is deployed, which reacts upon a new PR being opened in a given GitHub project. The bot is implemented using a service and event orchestration approach with {PRODUCT} implementation of the Serverless Workflow specification.
```
